clang/lib/Headers/CMakeLists.txt

```diff
@@ -623,6 +623,11 @@ install(
   EXCLUDE_FROM_ALL
   COMPONENT riscv-resource-headers)
 
+install(
```

Comment:
Aren't we also missing `sifive_files` from `add_header_target("riscv-resource-headers" "${riscv_files};${riscv_generated_files}")`?

Is there any reason to have `sifive_files`? Should we put sifive_vector.h in `riscv_files`?

Reply:
Yes, we are missing from add_header_target("riscv-resource-headers" "${riscv_files};${riscv_generated_files}").
I think we can just put sifive_vector.h in riscv_files, I'll have another patch for it, thanks!